src/systems/pref.cpp

```diff
@@ -87,14 +87,18 @@ enum class PrefId : u16 {
     IlBattleReadyBind = 67,
     FreecamToggleBind = 68,
     SavestateClearBind = 69,
+    TimerShowSubtick = 70,
+    TimerShowFramesave = 71,
```

Comment:
These conflict with your new IL Battle prefs in your other PR :(

Reply:
Updated!